doc/Overview/Mvux/Advanced/Commands.md

```diff
@@ -0,0 +1,209 @@
+
+        For example:
+
+        ```csharp
+        public IFeed<int> CounterValue => ...
+
+        public void ResetCounter(int counterValue)
+        {
+            ...
+        }
+        ```
+
+        When the command is executed, because the parameter name `counterValue` matches a feed name in the Model, this parameter will be materialized with the actual value from the Feed when this method is called on command execution.  
+        This behavior can be controlled and configured using the [`FeedParameter`](#feedparameter-attribute) and [`ImplicitFeedCommandParameter`](#implicitfeedcommandparameter-attribute) attributes.
+
+     - Can have one `CancellationToken` as its last parameter, but it's not mandatory.
+
+     > [!NOTE]  
+     > Name matching of command parameters to Feeds is case-insensitive.
+
+     > [!TIP]
+     > If you are using [Event Binding](https://learn.microsoft.com/en-us/windows/uwp/xaml-platform/x-bind-markup-extension#event-binding), you might need to opt out from command generation for those methods. See the next section on how to configure command generation.
+
+     ### Using attributes to control command generation
+
+     #### ImplicitCommands attribute
+
+     By default, implicit command generation is enabled by default. So that any method in the Model that matches the command criteria will be generated as a command.  
+     However, you may choose to switch this off for a specific method, class, or the entire assembly.
+
+     This can be achieved using the `ImplicitCommands` attribute. Here are some examples:
+
+     ```csharp
+     [assembly:ImplicitCommands(false)]
+     
+     [ImplicitCommands(true)]
+     public partial record MyModel();
+
+     [ImplicitCommands(true)]
+     public void MyCommand();
+     ```
+
+     These attributes can be used interchangeably to control command generation, for example, you may choose to disable the implicit feed command parameter altogether by using the attribute with the `assembly:` prefix. You can then opt-in for specific classes or methods. You can also switch off specific classes while opting-in certain methods by decorating them with this attribute as enabled. You can choose any combination to opt-in or out of implicit command generation.
+
+     #### FeedParameter attribute
+
+     You can explicitly match a parameter with a Feed even if the names don't match by decorating the parameter with the `FeedParameter` attribute:  
+     
+     ```csharp
+     public IFeed<string> Message { get; }
+     
+     public async ValueTask Share([FeedParameter(nameof(Message))] string msg)
+     {
+     }
+     ```
+
+     #### ImplicitFeedCommandParameter attribute
+
+     You can also opt in or out of implicit matching of Feeds and command parameters by decorating the current assembly or class with the `ImplicitFeedCommandParameters` attribute:  
```

Comment:
Should the be "opt-in or out"?

Verify that the hyphenation of this is consistent throughout the document.

Reply:
Does "opt-in or opt-out" sound better?

Reply:
Across the org hyphens are used more often for "opt-in" and "opt-out" than not.
I flagged this here because of inconsistencies in a single document.

In this instance, I'd personally opt for "You can also opt in to or out of the implicit matching of Feeds....." - I know this doesn't use a hyphen, but one would not be appropriate here.